### DISCUSSION TITLE
Customized Dawn Theme, where to find version number?

### DISCUSSION THREAD

**Discussion by fabianwohlfart** [Shopify/dawn#3437] [Q&A]
Hi guys,
I customized a dawn theme for a client.
In the panel it still says that my theme is in »Version 11« and that »Version 14« is up, despite that I adjusted the `theme_info` in the `settings_schema`. I cannot find »11.0.0« in my code, nor can I find a URL where this information should be pulled from.
How to solve (and remove) this?

![Bildschirmfoto 2024-04-25 um 12 34 11](https://github.com/Shopify/dawn/assets/15716389/499fdda4-b9b7-428c-b802-c1f8f4d2bc35)

**Comment by klemus** [comment]
Bump

**Comment by joshistoast** [comment]
At the top of your `settings_schema.json` file

**Reply by klemus** [reply]
Changing that does not remove the notice

**Comment by fabianwohlfart** [comment]
Hey guys, I am still struggling to remove this message.
Why does Shopify still think it's the dawn theme?
How can I remove this message – my client is also confused.
